edx_proctoring/api.py

```diff
@@ -1769,6 +1768,7 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
         student_view_template = 'proctored_exam/ready_to_submit.html'
 
     if student_view_template:
+        log.info("Proctoring context: %s", context)
```

Comment:
Same logging question as on [the other PR](https://github.com/edx/edx-platform/pull/13840#discussion_r85556862).  If no change is required there, none here either.  We'll see where that goes.